Makefile

```
@@ -104,6 +109,19 @@ setup: deps
 	@printf "# This file ensures the .github directory is tracked by Git\n# Remove this file once you add workflow files to the directory" > api-docs/.gitkeep
 
 
+.PHONY: generate
+generate:
```

checkmake (0.2.2)

[warning] Target body for "generate" exceeds allowed length of 5 (10).

(maxbodylength)